nixos/lib/make-disk-image.nix

```diff
@@ -70,6 +82,13 @@
 , # Disk image format, one of qcow2, qcow2-compressed, vdi, vpc, raw.
   format ? "raw"
 
+  # Whether to fix partitions GUIDs/UUIDs to arbitrary values.
```

Comment:
(More general task than for this line)

Be mindful of the confusing confusion that can happen with:

 - GPT Partition **Unique Identifier** [in the form of a UUID/GUID]
 - GPT Partition **Type** identifier [in the form for a GUID]
 - Filesystem **Unique Identifier** [filesystem dependent]

Can you ensure **every** instance of wording around those is properly identified to not be confused with one eachother? For example, never use the short form "partition guid" or worse, guid or uuid; but always *partition unique identifier*, *partition type identifier* and *filesystem unique identifier*?

Not necessarily for your own sake, but for the sake of everyone else re-reading this.

Reply:
You are totally right!

Reply:
Done.